sklearn/ensemble/tests/test_stacking.py

```diff
@@ -646,6 +653,85 @@ def fit(self, X, y):
         stacker.n_features_in_
 
 
+@pytest.mark.parametrize("stack_method", ["auto", "predict", "predict_proba"])
+@pytest.mark.parametrize("passthrough", [False, True])
+@pytest.mark.parametrize(
+    "X, y, is_multilabel",
+    [(X_multilabel, y_multilabel, True), (X_binary, y_binary, False)],
+    ids=["multilabel", "binary"],
```

Comment:
So it seems that you are testing multilabel and binary cases here.
I would not parametrize the test here and split into 2 separates test because the test does too much.